lua/nvim_lsp/r_language_server.lua

```diff
@@ -0,0 +1,71 @@
+local util = require 'nvim_lsp/util'
+local configs = require 'nvim_lsp/configs'
+
+configs.r_language_server = {
+  default_config = {
+    cmd = {"R", "--slave", "-e", "languageserver::run()"};
```

Comment:
```suggestion
    cmd = {"R", "--slave", "-e", "'languageserver::run()'"};
```

Reply:
It doesn't work here when I do this, so I left as it is.